PurchasesTests/SubscriberAttributes/PurchasesSubscriberAttributesTests.swift

```diff
@@ -142,10 +142,15 @@ class PurchasesSubscriberAttributesTests: XCTestCase {
 
     func testSetPushTokenMakesRightCalls() {
         setupPurchases()
+        let tokenData = Data("ligai32g32ig".data(using: .utf8)!)
+        let tokenString = (tokenData as NSData).asString()
 
-        Purchases.shared.setPushToken("asdf45asdg35asd5")
+        Purchases.shared.setPushToken(tokenData)
```

Comment:
What if data is nil?

Reply:
I'll add a test for it, but if data is nil, data.asString is no-op, and we call set attribute with nil which internally gets translated to `@""`

Reply:
test was already there: https://github.com/RevenueCat/purchases-ios/pull/192/files#diff-1bb60000f2b0d7699724860eb147241cL292
since the method takes a nullable NSData object and we're checking that it receives params correctly, I think another test might not be needed